src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders/ComplexTypeModelBinder.cs

```diff
@@ -403,5 +408,14 @@ protected virtual object CreateModel(ModelBindingContext bindingContext)
                 modelState.AddModelError(modelName, exception, bindingContext.ModelMetadata);
             }
         }
+
+        private bool HasDefaultConstructor(TypeInfo modelTypeInfo)
+        {
+            // The following check causes the ComplexTypeModelBinder to NOT participate in binding structs.
+            // - Reflection does not provide information about the implicit parameterless constructor for a struct.
+            // - Also this binder would eventually fail to construct an instance of the struct as the Linq's
```

Comment:
This isn't an "also" case. This is what makes it :ok: not to give all value types (which always have a parameterless constructor) a pass here.

Use consistent terminology: Stick with "parameterless constructor" instead of "default" everywhere.